guides/issuing-assets.md

```diff
@@ -39,7 +39,7 @@ In the simplest situations, you can issue assets from your everyday Stellar acco
 
 ### Requiring or Revoking Authorization
 
-Accounts have [several flags](concepts/accounts.md#flags) related to issuing assets. You’ll almost always want to make sure your issuing account has the [`AUTHORIZATION REVOCABLE` flag set](concepts/assets.md#revoking-access). This allows you to freeze your assets in another account.
+Accounts have [several flags](concepts/accounts.md#flags) related to issuing assets. You might want to make sure your issuing account has the [`AUTHORIZATION REVOCABLE` flag set](concepts/assets.md#revoking-access). This allows you to freeze your assets in another account.
```

Comment:
In my opinion _almost always_ is a good phase. Hovewer, I can imagine some use cases in which using `AUTHORIZATION REVOCABLE` flag is overcomplicating things or just shouldn't be used:
* Game assets: stars/points/etc. that usually have no real value outside a game.
* Some assets that should not be revocable and the owner of the asset **is able to keep the account safe from theft**. Ex. shares because issuer (company) should not be able to revoke them.
* Special case: limited assets (issuing account thresholds set to 0 after issuing an asset) so issuer can't revoke funds ([multisig](https://www.stellar.org/developers/guides/concepts/multi-sig.html#example-5-custom-currencies)).